libs/martech/helpers.js

```diff
@@ -338,6 +338,30 @@ function updateAMCVCookie(ECID) {
   }
 }
 
+function getUrl() {
+  const PAGE_URL = new URL(window.location.href);
+  const { host } = window.location;
+  const query = PAGE_URL.searchParams.get('env');
+  const url = 'https://edge.adobedc.net/ee/v2/interact';
+  const SLD = PAGE_URL.hostname.includes('.aem.') ? 'aem' : 'hlx';
+
+  /* c8 ignore start */
+  if (query || host.includes('localhost') || host.includes(`${SLD}.page`)
+    || host.includes(`${SLD}.live`)) {
+    return url;
+  }
```

Comment:
You can also do just check for `.live` or `.page` and get rid of the SLD part

Reply:
Done @mokimo